src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -53,8 +53,7 @@ class ExplorerButtonGroup extends React.Component {
         });
     }
     if (nextProps.totalCount !== this.props.totalCount
-      && nextProps.totalCount
-      && this.props.totalCount) {
```

Comment:
is this to fix the export to manifest btn disable bug? I sort of remembered that we put this implementation to prevent an error msg from showing in console? 🤔

Reply:
tested again but didn't see any errors. i guess my memory is not right. Anyway, pls ignore this and sorry for the confusions 🤦‍♂